### PR TITLE
UserGuidie: AutoSuggest and AutoExpense

### PR DESCRIPTION
**Pull request by SwampertX**
- [x] AutoSuggest
- [x] AutoExpense

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -56,6 +56,29 @@ a parameter which can be used as `n/mala at deck`.
...
+
+image::autosuggest_startup.png[]
+
+For example, you are new and would like to get some help! But your __shaky__ hands made you
+type `hlp` instead! Don't worry, we have a Edit-Distance-based algorithm that can predict your
+command. Isn't that cool?
+
+image::autosuggest_hlp.png[]
+
+Ahh okay! Seems that we can just kick start right away! How about recording that mala we ate The Deck
```
Comment (jessicax941) [inline docs/UserGuide.adoc]: _at_ the deck?*

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -56,6 +56,29 @@ a parameter which can be used as `n/mala at deck`.
...
+The first thing you will notice once you start typing in the command box, is AutoSuggest.
+This is an intelligent bot that helps you navigate through your finance journey with GuiltTrip()!
+
+Simply key in anything you like to start!!
+
+image::autosuggest_startup.png[]
+
+For example, you are new and would like to get some help! But your __shaky__ hands made you
+type `hlp` instead! Don't worry, we have a Edit-Distance-based algorithm that can predict your
+command. Isn't that cool?
```
Comment (jessicax941) [inline docs/UserGuide.adoc]: so cute

File: docs/UserGuide.adoc (hunk 1 above)
Comment (SwampertX) [reply docs/UserGuide.adoc]: NOPE I ATE THE DECK

Comment (jessicax941) [reply docs/UserGuide.adoc]: nAISE

**Pull request merged**